Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs

```diff
@@ -236,25 +240,8 @@ private string ProcessDocs(string docs)
                 result = result.Remove(metadataMatch.Index, metadataMatch.Index + metadataMatch.Length);
             }
 
-            // Need to do some cleaning
-            // Rework code tags
-            var regex = new Regex("```(xaml|xml|csharp)(?<code>.+?)```", RegexOptions.Singleline);
```

Comment:
@Vijay-Nirmal This Regex was the reason some of these documentation issues still worked in the old Sample App, it converted all Code Blocks into a different block type, which prevented Syntax Highlighting or anything else.

It has been removed so we can use it properly now.